### PR TITLE
Lucuma core

### PR DESCRIPTION
**Pull request by cquiroz**
This is a partial update of ITC to support the new lucuma-core
Only point source is supported and no emission lines can be accepted

### CODE REVIEW

File: modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala [hunk 1]
```diff
@@ -5,60 +5,41 @@ package lucuma.itc
...
 final case class ItcSourceDefinition(
-  profile:      SpatialProfile,
-  distribution: SpectralDistribution, // Switch to SpectralDistribution
-  norm:         MagnitudeValue,
-  units:        Either[MagnitudeSystem, SurfaceBrightness],
-  normBand:     MagnitudeBand,
-  redshift:     Redshift
+  profile:  SourceProfile,
+  normBand: Band,
+  redshift: Redshift
```
Comment (swalker2m) [inline modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala:22]: I thought ITC consumed radial velocity?

File: modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala [hunk 2]
```diff
@@ -80,32 +61,168 @@ object ItcSourceDefinition {
...
+            brightnesses.get(s.normBand).map(_.units.serialized) match {
+              case Some("VEGA_MAGNITUDE")                  => Json.obj("MagnitudeSystem" -> Json.fromString("Vega"))
+              case Some("AB_MAGNITUDE")                    => Json.obj("MagnitudeSystem" -> Json.fromString("AB"))
+              case Some("JANSKY")                          => Json.obj("MagnitudeSystem" -> Json.fromString("Jy"))
+              case Some("W_PER_M_SQUARED_PER_UM")          =>
+                Json.obj("MagnitudeSystem" -> Json.fromString("W/m²/µm"))
+              case Some("ERG_PER_S_PER_CM_SQUARED_PER_A")  =>
+                Json.obj("MagnitudeSystem" -> Json.fromString("erg/s/cm²/Å"))
+              case Some("ERG_PER_S_PER_CM_SQUARED_PER_HZ") =>
+                Json.obj("MagnitudeSystem" -> Json.fromString("erg/s/cm²/Hz"))
```
Comment (swalker2m) [inline modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala:119]: Hmm, the ITC uses these unit strings?

**Review by swalker2m — APPROVED** [review]
I'm not sure I know enough about this to really comment, but it looks good.  :+1

File: modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala (hunk 2 above)
Comment (cquiroz) [reply modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala:119]: the old itc, yes

File: modules/itc/src/main/scala/lucuma/itc/ItcImpl.scala [hunk 3]
```diff
@@ -78,7 +78,8 @@ object ItcImpl {
                                constraints,
                                exposures
             ).asJson
-          L.info(s"ITC remote query ${json.noSpaces}") *>
+          println(json.spaces2)
```
Comment (cquiroz) [inline modules/itc/src/main/scala/lucuma/itc/ItcImpl.scala:81]: ups

File: modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala (hunk 1 above)
Comment (cquiroz) [reply modules/itc/src/main/scala/lucuma/itc/ItcSourceDefinition.scala:22]: it gets converted to Redshift

**Pull request merged**